{
    "error": "An error occurred: HTTPSConnectionPool(host='mangatak.com', port=443): Max retries exceeded with url: /manga/6935974387-im-stuck-on-the-same-day-for-a-thousand-years (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001AD9534A6C0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))"
}